game/src/game/select_song.rs

```diff
@@ -124,37 +131,34 @@ pub(crate) fn select_song(
             }
 
             match event {
-                Event::KeyDown {
-                    // if user press right key, then song menu moves to right for specific distance
-                    keycode: Some(Keycode::Right),
-                    repeat: false,
-                    ..
-                } => {
-                    if moving_direction == MovingDirection::Stop {
-                        // to prevent changing direction when moving
-                        moving_direction = MovingDirection::Right;
-                        last_key_press_time = Instant::now();
-                    }
-                }
                 Event::KeyDown {
                     // if user press right key, then song menu moves to left for specific distance
-                    keycode: Some(Keycode::Left),
+                    keycode: Some(Keycode::L),
```

Comment:
Delete L/K key processing code, it's duplicated.

> Change Keyboard input to K and L (didn't delete keyboard input because, there's also keyboard input when selecting YES or NO at the deciding to do tutorial screen)

That's because K and L keys are fallback janggu keys when `--controller-port` argument is not given. `read_janggu_keys` method automatically handles that.